cmd/elkey/generate.go

```diff
@@ -0,0 +1,196 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/bls"
+
+	"github.com/Layr-Labs/eigensdk-go/crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	DefaultKeyFolder = "keys"
+
+	PasswordFile      = "password.txt"
+	PrivateKeyHexFile = "private_key_hex.txt"
+)
+
+var (
+	KeyTypeFlag = &cli.StringFlag{
+		Name:     "key-type",
+		Usage:    "key type to create (ecdsa or bls)",
+		Required: true,
+	}
+	NumKeysFlag = &cli.IntFlag{
+		Name:     "num-keys",
```

Comment:
should this be `max-num-keys` instead? What if `private_key_hex.txt` has fewer than `num-keys` lines?

Reply:
`num-keys` here is how many key we want to generate - we create all the files here

Reply:
But it reads the private keys from that hex file right?

Reply:
no no - this utility just create `num-keys` - new keys

Reply:
Ahh I see. I was misunderstanding how this is used.